client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -351,6 +351,10 @@
 	pointer-events: none;
 }
 
```

Comment:
This very specific CSS is needed because this task is effectively disabled/non-clickable, but we want it to show as enabled.